_headers

```diff
@@ -1,3 +0,0 @@
-/assets/logos/*
```

Comment:
The `_headers` file is picked up by Netlify and should still work after we get rid of jekyll. I don't know if this cache control stuff is necessary though

Reply:
@rwfeather - Apologies that I'm not super well versed in the implications. Do we need to do anything more to address, in this PR or in a follow-on? If a follow-on is warranted, can you log an issue for that action? Thanks!

Reply:
@rwfeather what AJ said? I think you're just calling this out but that sounds fine to me.

Reply:
I made these comments when I was reviewing it as Alex's PR. Just some files I noticed we might not want to delete. I'll be adding a commit that reverts that.